### ISSUE TITLE
cmints --example => error 

### ISSUE THREAD

**Issue by rdehuyss** [Manvel/cmints#169]
When running cmints --example and then start, I receive the following error:

```
ejs:8
    6|     <h2>{i18n-in-mind[Main text] CMS and static site generator created with the internationalization in mind}</h2>
    7|     <p>
 >> 8|       <a <%-i18n.href("documentation")%> class="button">{get-started[Main button text] Get started}</a>
    9|     </p>
    10|   </div>
    11| </section>

i18n.href is not a function
```

Am I doing something wrong?

**Comment by Manvel** [comment]
@rdehuyss No you are not :) Should work now.

I just didn't get chance to update the example project to catchup with the latest changes of the [i18n.href deprecation](https://github.com/Manvel/cmints/commit/f219b83264aea186281f8e007fbc57eac36e03e5). Currently it's functionality will be happen automagically as soon a relative link is used (Will also update documentation [soon](https://github.com/Manvel/cmints-website/issues/95)). 

BTW the example project currently is just modified version of [cmints-website](https://github.com/Manvel/cmints-website), basically same without analytics script.

The example project also will be changed soon, so something more simple. You can already give a try for it for [single page website](https://github.com/Manvel/cmints-single-lang-starter) and [multi language website](https://github.com/Manvel/cmints-multi-lang-starter) accordingly.

There is a [quick start guide](https://cmints.io/en/quick-start), but it's not final because that's currently only addressing single language website.

Note: Also for starter projects there is no need to install CMintS globally anymore, same actually also true for example projects, if you fork the repo, commands below will help you to get a quick start:
```
npm i 
npm run dev
```

I'm planing to make a full release in about 1 month from now, after that things should be more stable and documentation complete.

Please let me know if you might have any question.

Cheers!

**Comment by rdehuyss** [comment]
Great, works like a charm now.

Do you also have a UI for editing content already or is that on the roadmap?

**Comment by Manvel** [comment]
@rdehuyss That's a tough question :)

## Short answer
Nothing on the roadmap yet, the only thing currently in the roadmap is to finish documentation, do bug fixes if they appear and make a release of V1.0 and think about other stuff after that.

## Long Answer
If you would like to add something like an online IDE to manage your project, I would suggest for now to try going with Gitlab/Github online editors/[IDEs](https://docs.gitlab.com/ee/user/project/web_ide/), using it with [draft pages](https://cmints.io/en/documentation/pages#draft-pages).  You might have noticed Edit button in the [cmints.io documentation](https://cmints.io/en/documentation) pages and I basically using it for quick fixes, you might be required to fork the repo first if you are not a collaborator to the project, but people with editing rights will be navigated to the one page below:

<img width="1135" alt="screen shot 2019-01-29 at 2 36 57 pm" src="https://user-images.githubusercontent.com/716969/51912128-d3344300-23d3-11e9-89cb-df096dccd7c7.png">

For non registered collaborators maybe redirecting to [similar page](https://github.com/Manvel/cmints-website/blob/master/pages/documentation/getting-started/index.md) is better, but here the problem is that Github and Gitlab MD parsers will screw up the formatting of the Front Matter(well same true for the Preview mode of edit). It's not an idea solution, but in future I hope to find something better.

If you also integrate that with the Netlify, you can instantly publish changes and this might be something you are looking for.

If you are referring to the Content editing for translators, something like [pontoon](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localizing_with_Pontoon#In-context) does, that's something I would definitely consider for ease of translation. In fact Crowdin also have something called [in-context localization](https://support.crowdin.com/in-context-localization/), both are something I plan to look into in future and see how easy would it be to integrate into CMintS.

Hope this is helpful.
